src/components/flowchart/icon.js

```diff
@@ -30,16 +22,11 @@ const paths = {
 export default node => {
   const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
 
-  const iconSizes = {
-    data: 17,
-    task: 18,
-    parameters: 22
-  };
-  const iconSize = iconSizes[node.type];
+  const iconSize = 24;
```

Comment:
Is the icon size bigger now?

Reply:
![image](https://media.giphy.com/media/SVgKToBLI6S6DUye1Y/giphy.gif)

I've standardized the icons so the original viewbox of each SVG is 24x24, and they fill their containers more consistently and in a way that's expected. Also, the new icons have more negative space (i.e. they're less square, and more 'liney'), so they don't look as big even when they're displayed at the same size. So I've set them all to the same size and viewbox now, and increased their display size a touch, but they shouldn't appear to be much bigger. Compare:
![image](https://user-images.githubusercontent.com/1155816/66850842-813b4980-ef71-11e9-816e-1fc22c14aa9f.png) ![image](https://user-images.githubusercontent.com/1155816/66850860-87312a80-ef71-11e9-872a-9707d9cc29ed.png)